### PR TITLE
py-pint-xarray: new package

### PR DESCRIPTION
**Pull request by heerener**

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed both version sha256.

**Comment by tldahlgren** [comment]
Added python gurus as reviewers.

File: var/spack/repos/builtin/packages/py-pint-xarray/package.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+class PyPintXarray(PythonPackage):
+    """A convenience wrapper for using pint with xarray"""
+
+    homepage = "https://github.com/xarray-contrib/pint-xarray"
+    pypi = "pint-xarray/pint-xarray-0.2.1.tar.gz"
+
+    version("0.3", sha256="3545dfa78bee3f98eba29b8bd17500e3b5cb7c7b03a2c2781c4d4d59b6a82841")
+    version("0.2.1", sha256="1ee6bf74ee7b52b946f226a96469276fa4f5c68f7381c1b2aae66852562cb275")
+
+    depends_on("py-setuptools", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pint-xarray/package.py]:
```suggestion
    depends_on("py-setuptools@42:", type="build")
```

File: var/spack/repos/builtin/packages/py-pint-xarray/package.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+class PyPintXarray(PythonPackage):
+    """A convenience wrapper for using pint with xarray"""
+
+    homepage = "https://github.com/xarray-contrib/pint-xarray"
+    pypi = "pint-xarray/pint-xarray-0.2.1.tar.gz"
+
+    version("0.3", sha256="3545dfa78bee3f98eba29b8bd17500e3b5cb7c7b03a2c2781c4d4d59b6a82841")
+    version("0.2.1", sha256="1ee6bf74ee7b52b946f226a96469276fa4f5c68f7381c1b2aae66852562cb275")
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pint-xarray/package.py:17]:
```python
depends_on("python@3.8:", when="@0.3:", type=("build", "run"))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**